en/docs/fragments/ask-password.md

```diff
@@ -0,0 +1,96 @@
+
+3.  Expand the **User Onboarding** tab and configure the **Ask password code expiry time** field. Click **Update** to save changes.
+
+4.  In the same **User Onboarding** tab and select **Enable User Email
+    Verification**. Click **Update** to save changes.
+                
+    ![](../../../assets/img/fragments/resident-idp-ask-password-configs.png) 
+    
+    You can also configure the above configurations via the configuration
+    files. 
+    
+
+    ??? note "Click to see how to configure this through the configuration files" 
+        <a name="file-based-config"></a>
+        Make sure the following configuration is added
+        to the ` <IS_HOME>/repository/conf/deployment.toml ` file to set the
+        confirmation URL valid time period in **minutes**.  
+        The confirmation link that is provided to the user to set the
+        password is invalid after the time specified here has elapsed.
+
+        ``` toml
+        [identity_mgt.user_onboarding]
+        enable_email_verification = true
+        verification_email_validity = "1440"
+        lock_on_creation=true
+        [identity_mgt] 
+        email_sender= "internal"
+        [identity_mgt.user_onboarding]
+        ask_password_email_validity = "1440"
+        password_generator = "org.wso2.carbon.user.mgt.common.DefaultPasswordGenerator"
+        ```
+    
+    !!! info "Configure Ask Password Feature for tenants" 
+        These properties can be enabled for each tenant at tenant creation by
+        adding the corresponding configuration to the `
+        <IS_HOME>/repository/conf/deployment.toml` file as explained in the previous section. 
+            
+        Optionally, you can log in to the Management Console as a tenant
+        admin and change the **Resident IDP** configurations accordingly to enable this feature for a specific tenant.
+
+5.  Additionally, if you are adding users via the management console, to
+    enable "Ask password" option in the
+    Management Console the following property needs to be added to the `
+    <IS_HOME>/repository/conf/deployment-toml` file.
+
+    ``` toml
+    [identity_mgt.user_onboarding]
+    ask_password_from_user= true
+    ```
+
+6.  Enable the email sending configurations of the WSO2 Identity Server
+    as explained [here](../../../deploy/configure-email-sending/).
+      
+    !!! tip 
+        The email template used to send this email notification is
+        the **AskPassword** template.
+    
+        You can edit and customize the email template. For more information
+        on how to do this, see [Customizing Automated Emails](../../../guides/tenants/customize-automated-mails).
+
+
+Follow the steps below to test the account creation using the password option.
+
+1.  Access the Management Console.
+
+2.  On the Main tab in the **Management Console**, click **Add** under
+    Users and Roles.
+
+3.  Click **Add new User.**
+
+    ![add-a-new-user](../../../assets/img/fragments/add-a-new-user.png)
```

Comment:
IMO it is good if we can have a fragment for these 3 steps as it will be commonly used.